core/backend/src/ECSqlStatement.ts

```diff
@@ -689,6 +689,13 @@ export interface ECSqlColumnInfo {
    */
   getPropertyName(): string;
 
+  /** Gets the name of the original property that the column data is from.
+   * > Other than getPropertyName(), this ignores aliases and allows getting the name
```

Comment:
Does `>` have some special meaning here and elsewhere?

Reply:
vscode indents that portion in the tooltip.
![image](https://user-images.githubusercontent.com/18402661/210350655-239706e9-e525-44a7-bf64-50681ccaa555.png)

Reply:
@williamkbentley what happens to these `>`s on the docs site? Do they show up verbatim? Indent? Something else?

Reply:
Looks like it's turning into annotations on the website. https://www.itwinjs.org/reference/core-backend/ecsql/ecsqlcolumninfo/getpropertyname/

Reply:
Thanks, so typical markdown behavior.
I don't see much reason to do that vs just having multiple lines of documentation like we do elsewhere.
If something should be called out as, e.g., an important deviation from expected behavior, the `@note` tag is appropriate. I don't think it's needed here.